CMakeLists.txt

```diff
@@ -99,7 +99,7 @@ if(DAWN_DOCUMENTATION)
   set(doc_packages Python3 Sphinx LaTeX Doxygen)
 endif()
 
-foreach(package bash Protobuf ccache clang-format ${doc_packages})
+foreach(package bash Boost Protobuf ccache clang-format ${doc_packages})
```

Comment:
we actively avoided boost in dawn so far. Do you think this was a bad decision or do you think this is necessary enough that we have to take the hit here?

I know that the non-empy checks are a little more verbose, but it seems to be worth it in my eyes. If we want to decrease verbosity, we can also create our own optional (similar to our own any)

Reply:
Just to archive here:

We discussed this and came to the conclusion that we use header only libraries if they are fine in the current state (i.e a re-implementation does not gain us a significant advantage)